app/controllers/photos_controller.rb

```diff
@@ -5,7 +5,9 @@ class PhotosController < ApplicationController
   # GET /photos
   # GET /photos.json
   def index
-    @photos = @album.photos
+    # @desc is set to true if it has 'true' string or false for different input if the desc parameter is stipulated.
+    @desc = params[:desc] == 'true' if params[:desc]
```

Comment:
You don't need to do if params[:desc]
If params[:desc] is nil then @desc will be false